cli/tools/fmt.rs

```diff
@@ -291,6 +305,16 @@ fn get_markdown_config() -> dprint_plugin_markdown::configuration::Configuration
     .build()
 }
 
+fn get_jsonc_config() -> dprint_plugin_jsonc::configuration::Configuration {
+  let global_config =
+    dprint_core::configuration::resolve_global_config(HashMap::new()).config;
+  dprint_plugin_jsonc::configuration::resolve_config(
```

Comment:
Maybe construct `dprint_plugin_jsonc::Configuration` manually so we don't have to import `dprint_core`?

Reply:
Yes I tried to do that but even it requires we import `dprint_core::configuration::NewLineKind` so I just went with this approach .

Reply:
@dsherret Is there something we can do about it? :)

Reply:
I'll add a configuration builder.

Reply:
@satyarohith I change the crate so it's called `dprint-plugin-json` now instead of ending with `jsonc`. Upgrade to 0.8.0 and you will be able to do `ConfigurationBuilder::new().deno().build()`

Reply:
Thank you! ❤️